cheddar/transactions.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+from .resources.transaction import Transaction, TransactionResponse
+
+
+class Transactions(object):
+    def __init__(self, client):
+        self.requestor = client.requestor
+        self.configuration = client.configuration
+
+    def all(self, page=None, limit=None, before=None, after=None):
+        filters = {}
+
+        if page:
+            filters['page'] = page
+
+        if limit:
+            filters['limit'] = limit
+
+        if before:
+            filters['created_before'] = before.strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Even though the output is different, I think it is easier on the eyes and more expected than a custom format. (However, I know that both will work.)

```suggestion
            filters['created_before'] = before.isoformat()
```

The same with `filters['created_after']` a couple of lines below.